src/main/external-resources/web/jquery.ums.js

```diff
@@ -80,8 +82,9 @@ function changeMargins() {
 						height : row_h + 'px',
 						maxWidth : cell_w + 'px',
 						maxHeight : row_h + 'px',
-                        borderTopRightRadius: '3px',//MODDED
```

Comment:
Hello @Nagahar, I understand your point of view, and that's the correct thing to do. That's why I modified **jquery.ums.js** and not **jquery.min.js**, jquery.ums.js has functions only related to ums, those are the functions that organize the tiles for the views(grid,dynamic and now the new list view).

Reply:
Ok, it seems like I was simply mislead by the name of the file then. I know these files were there from before you got involved, but I never studied the web interface much.

Do you have any idea, if this isn't a modified version of jquery itself, why this name has been chosen?

Reply:
It seems that @SubJunk might be the one to answer why this name was chosen (544bf6370621407abfb24015288851eb532fa55b).

Reply:
Well, I have seen a lot of jquery to tell it is custom made, but no, I can't be sure about it.
The file doesn't have a header comment usually seen in jquery.
Code isn't minified nor obfuscated.
Has only 12 methods (I added 1).
It is called directly from the html, not using jquery sentences.
And I have just searched git history and it seems SubJunk created the file 5 years ago and it doesn't look anything close to what it looks today.
So, I think the name was chosen just because it was called after jquery using a jquery function: $(document).ready(function() {/*code*/});

Reply:
@Nadahar Yes, it seems so.

Reply:
It's just a naming convention that is sometimes used when working on jQuery to easily distinguish which files require jQuery to run vs. regular JavaScript. I can see how it can be confusing. I wouldn't mind if it were renamed to just `ums.js`

Reply:
Thanks @SubJunk , at least I understand why now. Whether it's a good practice or not is a different discussion I guess.